Makefile

```diff
@@ -141,9 +140,15 @@ clean:
 	rm -f *_stringer.go	# generated by `go generate`
 	rm -f *_mock.go		# generated by `go generate`
 
-test: bindata
+test: build
 	./test.sh
 
+# create all targets for make tab completion
+test_suites=$(shell find test -name test-* -exec basename {} .sh \;)
```

Comment:
shouldn't this be test/test-*.sh with a maxdepth? (so we don't get the subtests?)

Reply:
true